lib/Runtime/Types/DeferredTypeHandler.h

```diff
@@ -580,8 +621,9 @@ namespace Js
     {
         if (!EnsureObjectReady(instance, DeferredInitializeMode_Default))
         {
-            return FALSE;
+            return TRUE;
```

Comment:
Please review other API that you changed the return value as well to make sure we are doing the right thing.